addons/udes_stock/models/stock_picking.py

```diff
@@ -456,6 +456,16 @@ def assert_not_lot_restricted(self):
                             )
                         )
 
+    def button_draft(self):
+        """
+        Button that passes the picking to draft state. Adding extra check on backend that should be
```

Comment:
This wording might more idiomatic:

A button that moves the picking to the draft state. Adds an extra check in the backend that should be
triggered for only one transfer at time and the state is in waiting or confirmed
when the button is clicked.